google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/SpannerPoolTest.java

```diff
@@ -65,6 +65,8 @@ public class SpannerPoolTest {
 
```

Comment:
Query : Do we have a test which fails when the new property is not added to the equals() method?

Reply:
We can take it as a separate exercise, as we'll need this across classes. Keeping the thread open for the time being.